.vscodeignore

```diff
@@ -1,16 +1,14 @@
-# Run 'vsce ls' to verify that the following lines exclude everything not needed
-# in the VSIX. It used to be possible to use a whitelisting technique here
-# but vsce@1.92.0 broke it (see https://github.com/microsoft/vscode-vsce/issues/576)
-
-/*
-!/dist
-/dist/*.map
-!/images
-!/snippets
-!/syntaxes
+**
+!dist/*.js
+!snippets/
+!images/*.png
+!images/*.svg
```

Comment:
Can we reduce the two lines above to just this?
```
!images/
```

It looks like installation.gif is the only one that youre 2-line approach excludes, and since that file is apparently no longer used I suggest you delete it as part of this PR.